### PR TITLE
fix: fixed dev deployment scripts

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

Please include a summary of the changes and the related issue.

Resolves #(issue)

### CODE REVIEW

**Review by bigglesandginger — APPROVED** [review]
Either go with the unset method or with the shift.

File: kubernetes/dev_deploy.sh [hunk 1]
```diff
@@ -1,145 +1,152 @@
...
-  *)
-    echo "dev_deploy.sh: illegal option"
-    ;;
+	-h | --help)
+		COMMAND="help"
+		;;
+	up)
+		COMMAND="up"
+		SERVICES=("$@")
+		unset "SERVICES[0]"
```
[the remark this answers is not in the document]
Comment (dgauldie) [reply kubernetes/dev_deploy.sh]: fixed

File: kubernetes/dev_deploy.sh [hunk 2]
```diff
@@ -1,145 +1,152 @@
...
-  *)
-    echo "dev_deploy.sh: illegal option"
-    ;;
+	-h | --help)
+		COMMAND="help"
+		;;
+	up)
+		COMMAND="up"
+		shift
+		SERVICES=("$@")
```
Comment (bigglesandginger) [inline kubernetes/dev_deploy.sh:10]: you don't need the parentheses - but equally they don't hurt

Comment (dgauldie) [reply kubernetes/dev_deploy.sh:10]: it was failing for me, if I didn't include them

**Pull request merged**